<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgScriptParameters</class>
 <widget class="QDialog" name="CQDlgScriptParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Parameters</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Script parameters</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QListWidget" name="qqParameterList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter properties</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqValue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>331</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqUnit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqPrivate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Parameter is private</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqPersistent">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Parameter is persistent</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="qqAddNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Add new parameter</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqDown">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../variousImageFiles.qrc">
      <normaloff>:/variousImageFiles/downArrow.png</normaloff>:/variousImageFiles/downArrow.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="qqUp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../variousImageFiles.qrc">
      <normaloff>:/variousImageFiles/upArrow.png</normaloff>:/variousImageFiles/upArrow.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>300</y>
     <width>451</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqAddNew</tabstop>
  <tabstop>qqParameterList</tabstop>
  <tabstop>qqValue</tabstop>
  <tabstop>qqUnit</tabstop>
  <tabstop>qqPrivate</tabstop>
  <tabstop>qqPersistent</tabstop>
  <tabstop>qqClose</tabstop>
 </tabstops>
 <resources>
  <include location="../variousImageFiles.qrc"/>
 </resources>
 <connections/>
</ui>
